cmake/FindCUBLAS.cmake

```diff
@@ -0,0 +1,32 @@
+# 
+# To find CUDA:
+#   1. Use `enable_language(CUDA)`
+#   2. Set `CMAKE_CUDA_COMPILER` if cuda in custom directory
+#
+# Imported Targets: 
+#   dlaf::cublas
+#
+cmake_minimum_required(VERSION 3.12)
+
+find_path(CUDA_TOOLKIT_INCLUDE cuda.h
```

Comment:
Shouldn't be checked if `cuda_runtime.h` and `cublas_v2.h` exists in the include directory?

Reply:
To my knowledge, they are always in the same directory as `cuda.h`.

Reply:
I agree but I was thinking that it is a check if they are available.
Anyway if the cmake practice is to just check one of the headers I'm fine with it.